projects/common/src/color/colors.ts

```diff
@@ -0,0 +1,6 @@
+export enum Colors {
+  Purple2_15pOpacity = '#f2d0f5',
```

Comment:
Can we just call it Purple2, since we call it the same in _color-palette.scss?

Reply:
An enum for Colors sounds awesome!

Reply:
aah. my bad. I was testing something and forgot to change this back.